CSS/styles.css

```diff
@@ -45,8 +45,6 @@ ul {
     opacity: .7;
     border-style: solid;
     border-width: 4px;
-    margin: 150px auto 10px auto;
```

Comment:
The margin styles here were being overwrite by the margin style that proceeded it.